src/config/ALIENFLIGHTNGF7/config.h

```diff
@@ -108,7 +108,8 @@
 #define MAG_ALIGN CW180_DEG_FLIP
 #define MAG_ALIGN_PITCH 1800
 #define MAG_ALIGN_YAW 1800
-//TODO #define MAG_SPI_DEVICE 3
+#define USE_SPI_MAG
+#define MAG_SPI_INSTANCE SPI
```

Comment:
SPI3?

Reply:
Thanks. Updated. It was the only board I did manually :face_in_clouds: